spec/components/stylesheet/importer_spec.rb

```diff
@@ -138,4 +138,38 @@ def compile_css(name)
 
   end
 
+  context "#import_color_definitions" do
+    let(:scss) { ":root { --custom-color: green}" }
+    let(:scss_child) { ":root { --custom-color: red}" }
+
+    let(:theme) { Fabricate(:theme).tap { |t|
```

Comment:
This is a style thing but in Ruby I find using `{` for nested blocks somewhat confusing. I would recommend something like this:

```ruby
let(:theme) do 
  Fabricate(:theme).tap do |t|
    t.set_field(target: :common, name: "color_definitions", value: scss)
    t.save!
  end
end
```